airflow/api_internal/endpoints/rpc_api_endpoint.py

```diff
@@ -163,6 +163,12 @@ def log_and_build_error_response(message, status):
 
 def internal_airflow_api(body: dict[str, Any]) -> APIResponse:
     """Handle Internal API /internal_api/v1/rpcapi endpoint."""
+    content_type = request.headers.get("Content-Type", "")
+    if content_type != "application/json":
+        raise PermissionDenied("Expected Content-Type: application/json")
+    accept = request.headers.get("Accept", "")
+    if accept != "application/json":
+        raise PermissionDenied("Expected Accept: application/json")
```

Comment:
nit: there is no need to set default values?

```suggestion
    content_type = request.headers.get("Content-Type")
    if content_type != "application/json":
        raise PermissionDenied("Expected Content-Type: application/json")
    accept = request.headers.get("Accept")
    if accept != "application/json":
        raise PermissionDenied("Expected Accept: application/json")
```

Reply:
Yeah. Why not :)